### PR TITLE
react item lists in monitoring

### PR DESCRIPTION
**Pull request by petrjasek**
- [x] moved react part into its own file
- [x] handle highlights with react
- [x] put menu out of item dom so that it can be used withing scrolled areas
- [x] use react lists in monitoring
- [x] update react views with changes to media-box-list
- [x] fix multi select in monitoring
- [x] fix/implement add to package
- _remove old media-boxes code_ - will do this one later

### CODE REVIEW

**Comment by ioanpocol** [comment]
Please rebase

File: scripts/superdesk-search/react.js [hunk 1]
```diff
@@ -0,0 +1,1310 @@
...
+                            meta.push(React.createElement('dd', {key: 6}, moment(item.expiry).fromNow()));
+                        }
+
+                        var info = [];
+                        var flags = item.flags || {};
+
+                        info.push(React.createElement(
+                            'h5',
+                            {key: 1},
+                            item.slugline || item.type
```
Comment (naomiaro) [inline scripts/superdesk-search/react.js]: hey, I keep fixing this to be headline :D, now it's back again.

**Comment by akintolga** [comment]
:+1:

**Comment by petrjasek** [comment]
ping @superdesk/developers

**Comment by petrjasek** [comment]
not sure what else should be done here? @superdesk/developers

**Comment by ioanpocol** [comment]
please rebase

**Pull request merged**